src/styles/stars.scss

```diff
@@ -0,0 +1,30 @@
+.product-card__rating {
+  text-wrap: nowrap;
```

Comment:
The property `text-wrap` is not valid in CSS. You might be looking for `white-space: nowrap;` to prevent text from wrapping.